src/jenkins_job_insight/reportportal.py

```
@@ -398,27 +411,40 @@ def push_classifications(
             except _requests.exceptions.HTTPError as exc:
                 status = exc.response.status_code if exc.response is not None else None
                 detail = ""
+                response_body = ""
                 if exc.response is not None:
+                    response_body = exc.response.text
                     try:
                         rp_body = exc.response.json()
                         detail = rp_body.get("message", "")
                     except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)